pkg/controllers/systemd-etcd.go

```diff
@@ -0,0 +1,154 @@
+}
+
+func (s *ExtEtcdService) Run(ctx context.Context, ready chan<- struct{}, stopped chan<- struct{}) error {
+	defer close(stopped)
+
+	wd, err := os.Getwd()
+	if err != nil {
+		return fmt.Errorf("%s failed to get workdir: %v", s.Name(), err)
+	}
+
+	args := []string{
+		"--uid=root",
+		"--scope",
+		// Causes microshift-etcd to go away when MicroShift goes away (gracefully or otherwise)
+		"--property", "BindsTo=microshift.service",
+		"--unit", "microshift-etcd",
+		"/usr/bin/microshift", "etcd",
+	}
+	// TODO alray - Remove
+	// Our current etcd invocation from the CEO.
+	/*
+		exec ionice -c2 -n0 etcd \
+		--logger=zap \
+		--log-level=${VERBOSITY} \
+		--experimental-initial-corrupt-check=true \
+		--initial-advertise-peer-urls=https://${NODE_NODE_ENVVAR_NAME_IP}:2380 \
+		--cert-file=/etc/kubernetes/static-pod-certs/secrets/etcd-all-certs/etcd-serving-NODE_NAME.crt \
+		--key-file=/etc/kubernetes/static-pod-certs/secrets/etcd-all-certs/etcd-serving-NODE_NAME.key \
+		--trusted-ca-file=/etc/kubernetes/static-pod-certs/configmaps/etcd-serving-ca/ca-bundle.crt \
+		--client-cert-auth=true \
+		--peer-cert-file=/etc/kubernetes/static-pod-certs/secrets/etcd-all-certs/etcd-peer-NODE_NAME.crt \
+		--peer-key-file=/etc/kubernetes/static-pod-certs/secrets/etcd-all-certs/etcd-peer-NODE_NAME.key \
+		--peer-trusted-ca-file=/etc/kubernetes/static-pod-certs/configmaps/etcd-peer-client-ca/ca-bundle.crt \
+		--peer-client-cert-auth=true \
+		--advertise-client-urls=https://${NODE_NODE_ENVVAR_NAME_IP}:2379 \
+		--listen-client-urls=https://${LISTEN_ON_ALL_IPS}:2379,unixs://${NODE_NODE_ENVVAR_NAME_IP}:0 \
+		--listen-peer-urls=https://${LISTEN_ON_ALL_IPS}:2380 \
+		--metrics=extensive \
+		--listen-metrics-urls=https://${LISTEN_ON_ALL_IPS}:9978 ||  mv /etc/kubernetes/etcd-backup-dir/etcd-member.yaml /etc/kubernetes/manifests
+	*/
+	// args = []string{
+	// 	"--uid=root",
+	// 	"--scope",
+	// 	"--property", "BindsTo=microshift.service",
+	// 	"--unit", "microshift-etcd",
+	// 	"/home/microshift/etcd/bin/etcd",
+	// 	fmt.Sprintf("--name=%v", s.NodeName),
+	// 	fmt.Sprintf("--data-dir=%v", s.DataDir),
+	// 	"--logger=zap",
+	// 	"--experimental-initial-corrupt-check=true",
+	// 	fmt.Sprintf("--initial-advertise-peer-urls=%v", s.InitialUrl),
+	// 	fmt.Sprintf("--cert-file=%v", s.CertFile),
+	// 	fmt.Sprintf("--key-file=%v", s.KeyFile),
+	// 	fmt.Sprintf("--trusted-ca-file=%v", s.TrustedCAFile),
+	// 	"--client-cert-auth=true",
+	// 	// TODO alray - we might not need peers since we're single node
+	// 	fmt.Sprintf("--peer-cert-file=%v", s.PeerCertFile),
+	// 	fmt.Sprintf("--peer-key-file=%v", s.PeerKeyFile),
+	// 	fmt.Sprintf("--peer-trusted-ca-file=%v", s.PeerTrustedCAFile),
+	// 	"--peer-client-cert-auth=true",
+	// 	fmt.Sprintf("--advertise-client-urls=%v", s.AdvertiseClientUrls),
+	// 	fmt.Sprintf("--listen-client-urls=%v", s.ListenClientUrls),
+	// 	"--metrics=extensive",
+	// 	fmt.Sprintf("--listen-metrics-urls=%v", s.ListenMetricsUrls),
+	// }
```

Comment:
why is this still here?

Reply:
Removed the unneeded information, but kept the todo since the args there are what we should be using once we have the etcd binary.